dataprofiler/plugins/__init__.py

```diff
@@ -0,0 +1,34 @@
+import importlib
+import os
+
+from dataprofiler.plugins.decorators import plugin_decorator, plugins_dict
```

Comment:
inside we use relative:
`from .decorators import plugin_decorator, plugins_dict`